format/format.go

```diff
@@ -854,7 +854,7 @@ func (w *writer) writeComprehension(open, close byte, term *ast.Term, body ast.B
 }
 
 func (w *writer) writeComprehensionBody(open, close byte, body ast.Body, term, compr *ast.Location, comments []*ast.Comment) []*ast.Comment {
-	var exprs []interface{}
+	var exprs = make([]interface{}, 0, len(body))
 	for _, expr := range body {
 		exprs = append(exprs, expr)
```

Comment:
Could use copy()?

Reply:
The naive version `copy(exprs, body)` causes type errors, but I expect if I do some clever type conversion, I could maybe launder the `ast.Body` into being a `[]interface{}`.

Reply:
Aaah yeah. I've forgotten about that; it actually does something to the elements. Ignore me 😅